libraries/chain/get_config.cpp

```diff
@@ -52,6 +52,7 @@ fc::variant_object get_config()
    result[ "GRAPHENE_DEFAULT_MAINTENANCE_SKIP_SLOTS" ] = GRAPHENE_DEFAULT_MAINTENANCE_SKIP_SLOTS;
    result[ "GRAPHENE_MIN_UNDO_HISTORY" ] = GRAPHENE_MIN_UNDO_HISTORY;
    result[ "GRAPHENE_MAX_UNDO_HISTORY" ] = GRAPHENE_MAX_UNDO_HISTORY;
+   result[ "GRAPHENE_MAX_NESTED_OBJECTS" ] = GRAPHENE_MAX_NESTED_OBJECTS;
```

Comment:
@srpatel19590 Why is the nested objects changed ?

Reply:
It is not required. I've already removed this in my next commit.